### PR TITLE
Feature Created Contributors Page.

### LINKED ISSUE #135: [Feature Request] Add a clickable contribution page that displays all members who have contributed to the project.
### Description

Hello @XanderRubio , i've read the roadmap and i saw this feature idea. I would like to try and make this if possible.
I was wondering if i have the same idea of how to implement this as you.
Adding a button next to the current GitHub button on the homepage. 
After clicking the button you will go to a new page where you see all the contributors. 
Like the screenshot in a grid but ofcourse with the names of the contributor under it and maybe a link to their github page?

-Marciano

### Screenshots

exampleContributorsGrid

### Checklist

- [X] I have checked for duplicate [issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues)
- [X] I have read the [Contribution Guidelines]((https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md))
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by MarcianoN**
## Issue Addressed
Closes #135 
Added a contributors page, ContributorsGrid component, and a style sheet for that component.
Also added routing to the page.

## Proposed Changes and Benefits

I have created a contributors page for the website. Each contributor will be shown as a clickable card that leads to
their GitHub profile.

## Screenshots

FeatureScreenShot

## Notes to Reviewers

I did use npm install react-router-dom to create the routing.

### Checklist

- [X] I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
- [X] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior.
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
  - `Fix` - bug fixes.
  - `Docs` - fixing typos and adding new content.
- [X] I have listed my proposed changes and their benefits.
- [X] I have linked the corresponding issue in the Issues section.

### CODE REVIEW

**Comment by XanderRubio** [comment]
> ## Issue Addressed
> Closes #135 Added a contributors page, ContributorsGrid component, and a style sheet for that component. Also added routing to the page.
> 
> ## Proposed Changes and Benefits
> I have created a contributors page for the website. Each contributor will be shown as a clickable card that leads to their GitHub profile.
> 
> ## Screenshots
> FeatureScreenShot
> 
> ## Notes to Reviewers
> I did use npm install react-router-dom to create the routing.
> 
> ### Checklist
> * [x]  I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
> * [x]  The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
>   
>   * `Feature` - adds new or amends existing user-facing behavior.
>   * `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
>   * `Fix` - bug fixes.
>   * `Docs` - fixing typos and adding new content.
> * [x]  I have listed my proposed changes and their benefits.
> * [x]  I have linked the corresponding issue in the Issues section.


Wow, I haven't looked at the preview deployment link, and I love how the screenshot turned out of your design. Very professional looking! Clapping Hands

File: src/Components/ContributorsGrid/ContributorsGrid.jsx
Comment (XanderRubio) [inline src/Components/ContributorsGrid/ContributorsGrid.jsx:1]: Nice and well done with using the `getDifferentColors ` function to generates a random RGBA color for the Contributors" title and the boxShadow of each contributor card.

File: src/index.js
Comment (XanderRubio) [inline src/index.js:1]: Excellent, and congrats, Marciano, on being the first to create a new route for the Before I Die Code!!!! You have created another great starting point for the project to become continually enhanced and professional for developers worldwide, creating more opportunities to enhance the code. Thank you!

**Review by XanderRubio — APPROVED** [review]
Awesome job, @MarcianoN! Code can always be refined and structured with various naming conventions, and I appreciate the effort and time you put into starting this component and creating a polished Contibutors page. Your implementation and execution were top-notch. I'll be merging the changes once I fix a merge conflict issue, and I'll also be sending you an invitation to join the Before I Die Code organization later on once I have that set-up. As I want contributors to be able to participate in the GitHub discussions feature and contribute/collaborate more effectively. Thank you again for your hard work!

**Comment by MarcianoN** [comment]
Thanks for the kind words, @XanderRubio !
I had alot of fun with making this page and I learned a lot!
I hope I'll be able to do more contributions like this in the future or even more difficult ones when I 
get better at coding!

**Comment by XanderRubio** [comment]
> Thanks for the kind words, @XanderRubio ! I had alot of fun with making this page and I learned a lot! I hope I'll be able to do more contributions like this in the future or even more difficult ones when I get better at coding!

You are welcome to contribute to this project by suggesting features and improvements that interest you 😇. Additionally, I had to switch to merging your work into the `BeforeIDieCode:feature-contributor-page` to fix a conflict and will now merge it into the main branch. The process can become complicated quickly, and it is best to merge your feature work into the feature branch of the main if it exists (though it may not always be the case).

**Comment by XanderRubio** [comment]
This pull request is for issue #135 and will be closed one the feature is merged into the main branch

**Pull request merged**